models/exchange/binance/api.py

```diff
@@ -127,18 +127,15 @@ def getMakerFee(self, market: str='') -> float:
 
     def getTakerFee(self, market: str='') -> float:
         if market == '':
-            fees = self.getFees()
+            return DEFAULT_TAKER_FEE_RATE
         else:
             fees = self.getFees(market)
 
         if len(fees) == 0 or 'taker_fee_rate' not in fees:
             print (f"error: 'taker_fee_rate' not in fees (using {DEFAULT_TAKER_FEE_RATE} as a fallback)")
             return DEFAULT_TAKER_FEE_RATE
 
-        if market == '':
-            return fees
-        else:
-            return float(fees['taker_fee_rate'].to_string(index=False).strip())
+        return float(fees['taker_fee_rate'].to_string(index=False).strip()) * 100
```

Comment:
The default value is `0.001`, now I'm returning `0.001 * 100 = 0.1`

Reply:
This may effect lots of places and lead to extra division between fee calculations for different exchanges.

Why do we need this?

Reply:
The default maker and taker fees for Binance is `0.001`, so we either convert it to `0.10%` here or in every place we need this value we'd need to do `value * 100` to get the actual value.

Reply:
If fee rate is 0.1 and calculated transaction price is 9876 the fee calculation is:
9876*0.1/100 = 9.876
API already giving (0.1/100) part calculated so if we multiply by 100 here we will divide by 100 every time we need to calculate fee as you did at margin_calculator: 
sell_fee = round((sell_size /100) * sell_taker_fee, 8)

Reply:
I've fixed that now.